### PR TITLE
www.wiwo.de

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/120033

<details><summary>Screenshot:</summary>

Снимок экрана 2022-05-27 в 15 16 14

</details><br/>

### CODE REVIEW

**Review by BlazDT — CHANGES_REQUESTED** [review]
Please also add this rule to hide ads at bottom:
wiwo.de##div[tt-value="Serviceangebote unserer Partner"]

Also to Mobile app in Annoyances to hide black left-over at top:

`wiwo.de#$#body.c-app-install--hidden { padding-top: 0px !important; }`

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 1]
```diff
@@ -23,6 +23,7 @@
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
 m.kariyer.net##.download-app-wrapper
+wiwo.de#$#body.c-app-install--hidden { padding-top: 0px !important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/mobile-app.txt:26]: Not needed for desktop

Comment (BlazDT) [reply AnnoyancesFilter/sections/mobile-app.txt:26]: True, but desktop should not have that class in the first place.

Comment (zloyden) [reply AnnoyancesFilter/sections/mobile-app.txt:26]: We have directives at the end of the file :)

**Review by zloyden — APPROVED** [review]

**Review by BlazDT — APPROVED** [review]

**Pull request merged**